src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -195,7 +196,7 @@ const TimeEntryForm = props => {
     const today = moment().tz('America/Los_Angeles');
     const isDateValid = date.isValid();
     // Administrator/Owner can add time entries for any dates, and other roles can only edit their own time entry in the same day.
-    const isUserAuthorized = (canEditTimeEntry && canPutUserProfileImportantInfo) || !edit || today.diff(date, 'days') === 0
+    const isUserAuthorized = (canEditTimeEntry && canPutUserProfileImportantInfo) || canEditTimeEntryDescription || !edit || today.diff(date, 'days') === 0
```

Comment:
```suggestion
    const isUserAuthorized = (canEditTimeEntry && canPutUserProfileImportantInfo) || !edit || today.diff(date, 'days') === 0
```
isUserAuthorized looks like it's only about changing dates of time entries, not descriptions?